packages/react-reconciler/src/__tests__/useMutableSourceHydration-test.js

```diff
@@ -438,19 +408,17 @@ describe('useMutableSourceHydration', () => {
         dispatchAndSetCurrentEvent(arbitraryElement, mouseOverEvent);
 
         expect(Scheduler).toFlushAndYieldThrough([2]);
-
         source.value = 'two';
       });
     }).toErrorDev(
-      [
-        'Warning: An error occurred during hydration. ' +
-          'The server HTML was replaced with client content in <div>.',
-
-        'Warning: Text content did not match. Server: "1" Client: "2"',
-      ],
-      {withoutStack: 1},
+      'Warning: Text content did not match. Server: "1" Client: "2"',
     );
-    expect(Scheduler).toHaveYielded([2, 'a:two']);
     expect(source.listenerCount).toBe(1);
+    if (gate(flags => flags.enableSyncDefaultUpdates)) {
+      expect(Scheduler).toHaveYielded([2, 'a:two']);
+    } else {
+      // Not sure about this...
```

Comment:
@acdlite  Unsure why I needed to do this. Could you enlighten me please? :)

Reply:
`hydrateRoot` is synchronous whereas `root.render` is not